packages/ai-jsx/src/batteries/use-tools.tsx

```diff
@@ -208,7 +199,11 @@ export interface UseToolsProps {
  */
 export async function* UseTools(props: UseToolsProps, { render }: RenderContext) {
   try {
-    const rendered = yield* render(<UseToolsFunctionCall {...props} />);
+    const rendered = yield* render(<UseToolsFunctionCall {...props} />, {
```

Comment:
I'm pretty sure this can lead to flickering, need to rework this (and/or punt until better error boundaries)